recipes/minikube_setup/configure.py

```
@@ -0,0 +1,148 @@
+        lines = output.split("\n")
+        gpu_count = len(lines)
+        driver_version = lines[0].split(",")[0].strip()
+        check_output(["nvidia-smi", "-L"], stderr=DEVNULL)
+
+        return True, f"NVIDIA driver {driver_version} loaded, {gpu_count} GPU(s) found"
+
+    except FileNotFoundError:
+        return False, "nvidia-smi not found — NVIDIA driver may not be installed"
+    except CalledProcessError as e:
+        return False, f"nvidia-smi failed with exit code {e.returncode}"
+    except Exception as e:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)